dist/index.js

```diff
@@ -1754,7 +1755,11 @@ function getApps() {
             core.error(e);
         }
         return responseJson.items.filter(app => {
-            return (app.spec.source.repoURL.includes(`${github.context.repo.owner}/${github.context.repo.repo}`) && (app.spec.source.targetRevision === 'master' || app.spec.source.targetRevision === 'main' || app.spec.source.targetRevision === 'HEAD'));
+            return (app.spec.source.repoURL.includes(`${github.context.repo.owner}/${github.context.repo.repo}`)
```

Comment:
Extra `(` here isn't needed:
```suggestion
            return app.spec.source.repoURL.includes(`${github.context.repo.owner}/${github.context.repo.repo}`
```